tox.ini

```diff
@@ -0,0 +1,13 @@
+[tox]
```

Comment:
I don't know how to use `tox`, so some instructions in the README on how to run the included tests would be great!

Reply:
OK, I don't know much about it either, It just comes for free with the template. I will do some reading and or talk to Sierra. At the least, it can check compatibility between the project and several different versions of Python.

Reply:
`tox` is currently optional in this repo, but does have some advantages. I think BBOP will be making more use of it. added some notes to `MAINTAINERS.md` I still don't know a lot about it.